content/guides/authentication/jwt/without-sdk.md

```diff
@@ -16,52 +16,51 @@ alias_paths:
 
 # JWT without SDKs
 
-If you are not ready to use any of the official Box SDKs, or an SDK is not
-available in your language of choice, it is totally possible to use the Box APIs
-without them.
+This guide takes you through JWT authentication without using a Box SDK.
+JWT does not require end-user interaction and is designed to authenticate
+directly with the Box API. 
 
-This guide will take you through user authentication using JWT without the use
-of the Box SDKs. JWT authentication is designed for working directly with the
-Box API without requiring a user to redirect through Box to authorize your
-application.
+There are two ways you can verify an application's permissions:
 
-## Overview
+- using a public and private key pair
+- using a client id and client secret (Client Credentials Grant)
 
-To complete a JWT authorization the following steps need to be completed.
+Using a Client Credentials Grant is the easiest and fastest verification
```

Comment:
I'm not sure that we should call out "Using a Client Credentials Grant is the easiest and fastest verification method for prototyping or scripting against your Box enterprise", since it'll guide folks into using that method first. My reasoning here is that a key pair will create an app without a given expiration timeframe, like client cred grant, so we'll probably see better application integrations with the key pair method. I'd recommend dropping this sentence or giving more of a good point / bad point view of each type to provide all info.